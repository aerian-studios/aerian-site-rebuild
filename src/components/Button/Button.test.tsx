/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { Button } from "./index";

describe("Button", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <Button alternate={true} arrow={true}>
                    <a href="#">Hello</a>
                </Button>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
