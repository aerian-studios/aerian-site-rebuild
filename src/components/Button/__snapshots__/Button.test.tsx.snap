// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<span
  className="undefined"
>
  <a
    href="#"
  >
    Hello
  </a>
  <i
    className="fa"
  />
</span>
`;
